src/Tribe/Aggregator.php

```diff
@@ -128,13 +128,20 @@ private function __construct() {
 		// Remove aggregator records from ET
 		add_filter( 'tribe_tickets_settings_post_types', array( $this, 'filter_remove_record_post_type' ) );
 
-		// Notice users about expiring Facebook Token
+		// Notice users about expiring Facebook Token if oauth is enabled
+		add_action( 'plugins_loaded', array( $this, 'setup_notices' ), 11 );
+	}
 
-		/**
-		 * @todo  fb-reviewed-reactivation
-		 */
-		// tribe_notice( 'tribe-aggregator-facebook-token-expired', array( $this, 'notice_facebook_token_expired' ), 'type=error' );
-		// tribe_notice( 'tribe-aggregator-facebook-oauth-feedback', array( $this, 'notice_facebook_oauth_feedback' ), 'type=success' );
+	/**
+	 * Maybe show notices to users about expiring Facebook Tokens if oauth is enabled
```

Comment:
Kind of related to the comment on line 131: the method name sounds more general purpose to me than something that will/should only ever deal with FB stuff. Should we reword the comment to make it clear Facebook notices are merely its current concern, or else rename the method if the intent is to keep it Facebook-specific?